plugins/modules/ipadnsrecord.py

```diff
@@ -657,7 +753,7 @@
   loc_lon_dir:
     description: LOC Direction Longitude
     required: false
-    choices: ["E", "W"]
+    type: str
```

Comment:
Shouldn't we keep the choices "E" and "W", as only them should be used (RFC 1876)?